content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,212 @@
+
+### How to use the symbolic toolbox
+This toolbox is enabled in Matlab using the function `syms`. If you have an expression `x=2*a+b`, and you try to execute it in Matlab, you get an error message *undefined function or variable 'a'* as shown below:
+```Matlab
+x = 2*a + b
+```
+Output
+
+```Matlab
+Undefined function or variable 'a'.
+```
+When using the `syms` function, the variable `x` is saved without the error message. The idea here is, when using the sSymbolic toolbox, you first define the symbolic variables. Symbolic variables are the undefined variables in an equation. For our expression above, our symbols are `a` and `b`, so we first define these variables using the symbolic function `syms`.
+```Matlab
+syms a b
+```
+After defining the symbols and rerunning the code above, our variables are stored. We will have:
+```Matlab
+syms a b
+x = 2*a + b
+```
+Output:
+```Matlab
+x =
+ 
+2*a + b
+```
+
+### Solving Fourier series using the symbolic toolbox
+Let's say we have a Fourier transform shown below:
+
+$$\int e^{(\frac{-t}{2})}u(t) e^{-jwt}dt$$
+
+The symbol variables are t, w, T, and W, which we define by executing the command below:
+```Matlab
+syms t w T W
+```
+`w` is the angle theta.
+`T` is the time function.
+`W` is used to express the angular radians in the Fourier transform.
+
+After declaration of the symbol variables, you can write the Fourier transform as shown below:
+```Matlab
+f = int(exp(-t/2)*exp(-j*w*t), t, [0, inf])
+```
+In Matlab, `int` is used to mean integration. In the expression in Matlab above:
+`exp(-t/2)` is our equation which we are finding its Fourier transform. 
+`exp(-j*w*t)` is the basic function of the Fourier transform.
+`t` shows that we are differentiating with respect to time.
+`[0, inf]` shows the integration limits from `0` to `infinity`.
+
+> Note that we don't write `f(t)` when writing our equation in Matlab. It is because we already know that our function is a time function. Also, `t` is a symbol variable, so we do not write it in our expression.
+
+When the above program is executed, we get the output below:
+```Matlab
+f =
+ 
+2/(1 + w*2i) - (2*limit(exp(-t/2)*exp(-t*w*1i), t, Inf))/(1 + w*2i)
+```
+For formatting of this output to a user-friendly manner, we use a function pretty(). `pretty(f)` prints the symbolic expression `f` in a format that resembles type-set mathematical equations.
+```matlab
+ pretty(f)
+           /             /   t \              \
+           |    lim   exp| - - | exp(-t w 1i) | 2
+    2      \ t -> Inf    \   2 /              /
+-------- - --------------------------------------
+1 + w 2i                  1 + w 2i
+```
+Now, we need to get the values of `w` since we use them to plot the Fourier transform. To get the values, we execute the code below:
```

Comment:
Again transform or series? If it should be series, please rectify in any other part of the file.